crates/next-core/src/next_font_google/mod.rs

```diff
@@ -76,11 +79,27 @@ impl ImportMappingReplacement for NextFontGoogleReplacer {
                         r#"
                             import cssModule from "@vercel/turbopack-next/internal/font/google/cssmodule.module.css?{}";
```

Comment:
Why aren't we putting this into `@next/font/google/cssmodule.module.css`?

Reply:
These are all virtual assets and will never exist on disk at those locations. I figured it might be confusing or misleading to present them as if they did.

Reply:
Isn't the whole `@next/font` package virtual?

looking at https://github.com/vercel/next.js/blob/canary/packages/font/google/index.js